modules/distribution-zones/src/testFixtures/java/org/apache/ignite/internal/distributionzones/DistributionZonesTestUtil.java

```diff
@@ -18,29 +18,53 @@
 package org.apache.ignite.internal.distributionzones;
 
 import java.util.concurrent.CompletableFuture;
+import java.util.function.Consumer;
 import org.apache.ignite.internal.distributionzones.DistributionZoneConfigurationParameters.Builder;
+import org.apache.ignite.internal.schema.configuration.storage.DataStorageChange;
 
 /**
  * Utils to manage distribution zones inside tests.
  */
 public class DistributionZonesTestUtil {
+
     /**
      * Creates distribution zone.
      *
      * @param zoneManager Zone manager.
      * @param zoneName Zone name.
      * @param partitions Zone number of partitions.
      * @param replicas Zone number of replicas.
+     * @param dataStorageChangeConsumer Consumer of {@link DataStorageChange}, which sets the right data storage options.
      * @return A future, which will be completed, when create operation finished.
      */
     public static CompletableFuture<Integer> createZone(
-            DistributionZoneManager zoneManager, String zoneName, int partitions, int replicas) {
+            DistributionZoneManager zoneManager, String zoneName,
+            int partitions, int replicas, Consumer<DataStorageChange> dataStorageChangeConsumer) {
```

Comment:
Each parameter should start from a new line, in case of not enough characters for entire function definition in one line.